pkg/mgmt/volume/volume.go

```diff
@@ -246,3 +263,20 @@ func (c *VolController) processNextWorkItem() bool {
 
 	return true
 }
+
+func (c *VolController) transformDeviceVolError(err error) *apis.VolumeError {
+	volErr := &apis.VolumeError{
+		Code:    apis.Internal,
+		Message: err.Error(),
+	}
+	execErr, ok := err.(*volumeerror.Error)
+	if !ok {
+		return volErr
+	}
+
+	if strings.Contains(strings.ToLower(string(execErr.Err.Error())),
```

Comment:
Doesnt `Err.Error()` already return a string? Do we need to convert it again?

Reply:
resolved